fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -874,6 +877,14 @@ public long getMaxWriteLimitBytes() {
             return maxWriteLimitBytes;
         }
 
+        /**
+         * Exit with exception if this limit is exceeded (checked after each non-final iteration).
+         * @return time limit, in millisecond units.
```

Comment:
```suggestion
         * @return time limit in milliseconds
```

Same for the other javadocs referencing units. I appreciate that the units here are documented, though 👍

Reply:
Changed. 
(Though the comma was attributing a dramatic effect :)